ProcessLib/LIE/HydroMechanics/LocalAssembler/HydroMechanicsLocalAssemblerMatrixNearFracture-impl.h

```diff
@@ -75,9 +79,9 @@ void HydroMechanicsLocalAssemblerMatrixNearFracture<
 
     // levelset value of the element
     // remark: this assumes the levelset function is uniform within an element
-    auto const& fracture_props = *_process_data.fracture_property;
```

Comment:
`fracture_props` was set as member of the class as `_fracture_props`. However I could not found where  `_fracture_props` is initialized. May be I am wrong.

Reply:
`fracture_property` is initialized in CreateSmallDeformationProcess.cpp and moved to the process data. see https://github.com/ufz/ogs/blob/master/ProcessLib/LIE/SmallDeformation/CreateSmallDeformationProcess.cpp#L143

Reply:
I see that is initialized after creating its owner.